sherpa/astro/data.py

```diff
@@ -2102,6 +2104,9 @@ def get_analysis(self):
         Examples
         --------
 
+        >>> from sherpa.astro.io import read_pha
+        >>> pha = read_pha('sherpa/astro/datastack/tests/data/3c273.pi')
```

Comment:
It's going to be very annoying if we have to add imports before each example.

Reply:
We can do that once in the global setup (I need to check where it goes, but something like and ini or `conf.py` file)